region/brinstar/blue/Construction Zone.json

```diff
@@ -177,6 +214,23 @@
       "gModeRegainMobility": {},
       "note": "Shoot a Super at the wall, while the Geemer is moving vertically. If it is on the bottom of the shot blocks, it will not fall."
     },
+    {
+      "link": [2, 3],
+      "name": "Carry Shinecharge",
+      "entranceCondition": {
+        "comeInShinecharged": {
+          "framesRequired": 160
```

Comment:
Best I could get with room grinding was 165

Reply:
159 new best 💪Had to morph immediately on entry.
Also Morph is missing from requirements.

Reply:
Good catch, added Morph requirement now